binplace.targets

```diff
@@ -10,21 +10,35 @@
     <BinPlaceStuffDependsOn Condition="'$(IsReferenceAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceReferenceAssembly</BinPlaceStuffDependsOn>
     <BinPlaceStuffDependsOn Condition="'$(IsRuntimeAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceRuntimeAssembly</BinPlaceStuffDependsOn>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <BinplaceConfiguration Include="$(BuildConfiguration)">
+      <RefPath>$(RefPath)</RefPath>
```

Comment:
I can add a new property for this.  We can't yet change the value for $(RefPath) since that we don't have complete netstandard targeting pack.